app/controllers/api/photos_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::PhotosController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def update
+    if params[:image]
+      curr_photo = photo
+      curr_photo.image_url = params[:image]
+      curr_photo.save
+      return render json: {success: true, url: curr_photo.image.url}
+    else
+      return render json: {success: false}
+    end
+  end
+
+  def remove
+    curr_photo = photo
+    curr_photo.image = nil
+    curr_photo.save
+    return render json: {success: true}
+  end
+
+  private
+
+  def photo
+    @photo = Photo.find_by meta_id: :meta_id
+    if (!@photo)
+      @photo = Photo.new
+      @photo.meta_id = :meta_id
+    end
+    @photo
```

Comment:
i'd probably rewrite this to use instance variable memoization:
```rb
@photo ||= Photo.find_or_initialize_by(meta_id: :meta_id)
```

Reply:
But also, what is the function of the `:meta_id` symbol value here? Just a placeholder until dragonfly updates it?

Reply:
Since I already had the `:meta_id` for the scene, I thought that would be a useful thing to have so when I was replacing or removing the image, I could just find the image by the `:meta_id` to do so without having to store the uid in the JSON separately.